t/job-info/info_lookup.c

```diff
@@ -0,0 +1,81 @@
+/************************************************************\
+ * Copyright 2023 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
+#include <unistd.h>
+#include <jansson.h>
+#include <stdio.h>
+#include <flux/core.h>
+
+#include "src/common/libutil/log.h"
+
+int main (int argc, char *argv[])
+{
+    flux_t *h;
+    flux_future_t *f;
+    flux_jobid_t id;
+    json_t *keys;
+    int i;
+
+    if (argc < 3) {
+        fprintf (stderr, "Usage: info_lookup <jobid> <key> ...\n");
+        exit (1);
+    }
+
+    if (!(h = flux_open (NULL, 0)))
+        log_err_exit ("flux_open");
+
+    if (flux_job_id_parse (argv[1], &id) < 0)
+        log_msg_exit ("error parsing jobid: %s", argv[1]);
+
+    if (!(keys = json_array ()))
+        log_err_exit ("json_array");
```

Comment:
jansson functions don't claim to set errno so this should probably use `log_msg_exit()`.